### DISCUSSION TITLE
REPO_MIRROR_TLS_VERIFY default is disabled?

### DISCUSSION THREAD

**Discussion by jcstanaway** [quay/quay#1036] [General]
Any particular reason in this day & age that the default for this is False???

https://access.redhat.com/documentation/en-us/red_hat_quay/3.6/html/configure_red_hat_quay/config-fields-intro#config-fields-nested-repositories

**Comment by flavianmissi** [comment]
Hi @jcstanaway, I'm not sure the docs are up-to-date as the value is set to True on both [quay](https://github.com/quay/quay/blob/master/config.py#L534) and [config-tool](https://github.com/quay/config-tool/blob/master/pkg/lib/fieldgroups/repomirror/repomirror.go?plain=1#L14) sources'.
We'll double check this to make sure the docs match the actual defaults - thanks for bringing this to our attention.
